tests/components/bond/test_cover.py

```diff
@@ -1,23 +1,21 @@
 """Tests for the Bond cover device."""
+from datetime import timedelta
 import logging
 
 from bond import BOND_DEVICE_TYPE_MOTORIZED_SHADES
 
 from homeassistant import core
 from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
-from homeassistant.components.homeassistant import (
-    DOMAIN as HA_DOMAIN,
-    SERVICE_UPDATE_ENTITY,
-)
 from homeassistant.const import (
     ATTR_ENTITY_ID,
     SERVICE_CLOSE_COVER,
     SERVICE_OPEN_COVER,
     SERVICE_STOP_COVER,
 )
 from homeassistant.helpers.entity_registry import EntityRegistry
-from homeassistant.setup import async_setup_component
+from homeassistant.util import utcnow
 
+from ...common import async_fire_time_changed
```

Comment:
We prefer absolute imports for extra package imports.

Reply:
Got it, changed.